mergify_engine/queue/merge_train.py

```diff
@@ -797,6 +801,20 @@ async def _set_creation_failure(
                     source="draft pull creation error",
                 )
 
+    async def get_timeout(self) -> typing.Optional[datetime.timedelta]:
+        queue_name = self.initial_embarked_pulls[0].config["name"]
+        rules = await self.train.get_queue_rules()
+
+        if not rules or queue_name not in [rule.name for rule in rules]:
+            raise RuntimeError("Rules are missing from configuration")
```

Comment:
this is always called no matter the context and thus making the RuntimeError fails other tests like `test_create_pull_basic`, need to work on this

Reply:
I see, if that's only tests in TestTrainApiCalls, they should be updated to fix the configuration file.